.travis.yml

```diff
@@ -0,0 +1,30 @@
+# Copyright Intel Corp. All Rights Reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+dist: bionic
+language: minimal
+
+sudo: required
+services:
+  docker
+
+addons:
+  apt:
+    packages:
+
+env:
+  global:
+
+before_install:
+
+script:
+  # travis doesn't check out the branch itself but for our make to work we
+  # need a branch or alike. So check create a local one ...
+  - git checkout -b we-need-a-branch-for-below-to-work
+  # do integration tests for all available interpreters (in SGX_MODE=SIM mode,
+  # though, only ...)
+  # - gipsy/scheme
+  - PDO_INTERPRETER=gipsy make -C docker test
+  # - wawaka/wasm
+  - PDO_INTERPRETER=wawaka make -C docker test-env-setup # replace with test once integration tests also work with wawaka
```

Comment:
What is the new file? I am afraid I don't understand how travis works? (A demo might be helpful). Did we have travis before?

Reply:
nope, travis is the continuous integration (CI) tool.  It will will run on each push to a PR on the main repo and flag if the build or integration test fails (if you look at this PR, you will see a new check at the end of the PR related to travis).  This file defines what travis does.    Demo might be a good idea, though, maybe i could do that once i'm back in a TDS weekly meeting early january?  Bruno, though, also knows travis (we use it in FPC) so he could give you also demo ...